src/sentry/utils/sdk_crashes/sdk_crash_detection.py

```diff
@@ -30,6 +31,9 @@ def __init__(
 
     def detect_sdk_crash(self, event: Event) -> None:
 
+        if not features.has("organizations:sdk-crash-reporting", event.project.organization):
+            return
```

Comment:
I would check this feature flag in `post_process.sdk_crash_monitoring` before initiating the timer metric and the span, to reduce overhead and to not skew the metrics.